charts/bitcoind/templates/statefulset.yaml

```diff
@@ -111,6 +111,47 @@ spec:
             successThreshold: 3
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        {{- if .Values.descriptorImport.enabled }}
+        - name: descriptor-import
+          image: lncm/bitcoind:v24.0.1
+          command: ['/bin/sh']
+          args:
+          - '-c'
+          - | 
+            echo "# Waiting for bitcoind to finish syncing"
+            while ! bitcoin-cli getblockchaininfo | grep '"initialblockdownload": false'; do
+              sleep 30
+            done
+            echo "# Checking for the default wallet"
+            if ! bitcoin-cli listwallets | grep "default"; then
+              if [ -d /data/.bitcoin/signet/wallets/default ]; then
```

Comment:
The chart knows the network right? - so we we shouldn't need this conditional.

Reply:
now checking if loadwallet fails proceeds to create it